pypeit/par/pypeitpar.py

```diff
@@ -3505,7 +3523,7 @@ class ExtractionPar(ParSet):
 
     def __init__(self, boxcar_radius=None, std_prof_nsigma=None, sn_gauss=None,
                  model_full_slit=None, manual=None, skip_optimal=None,
-                 use_2dmodel_mask=None):
+                 use_2dmodel_mask=None, optimal_fwhm=None):
```

Comment:
This should be changed to a boolean use_user_fwhm, or something appropriate to indicate that automatic fwhm determination is being disabled, as we do for manual objects.

Reply:
Done